app/assets/json/translations/react/en.json

```diff
@@ -64,10 +64,12 @@
   "listings.reservedCommunityType.habitat": "Habitat Greater San Francisco",
   "listings.reservedCommunityType.senior": "Senior Building",
   "listings.reservedCommunityType.veteran": "Veterans Building",
-  "listings.unitTypes.1 BR": "1 BR",
-  "listings.unitTypes.2 BR": "2 BR",
-  "listings.unitTypes.3 BR": "3 BR",
-  "listings.unitTypes.SRO": "SRO",
+  "listings.unitTypes.1 BR": "1 Bedroom",
```

Comment:
I think we don't want these spaces in the keys, I'm not sure that it'll work with Phrase with the type of files we have set up.

It is a nice solution, so we could try it to see if it'll work OK.

Reply:
right, @christine-sfg and I agreed that it's funky. if this works as a first pass we could dogear it as something to come back to